src/basis/utils/assign.js

```diff
@@ -0,0 +1,34 @@
+// ES6 Object.assign polyfill
+// jscs:disable
+
+if (!Object.assign) {
+    Object.defineProperty(Object, 'assign', {
```

Comment:
I think that Object.assign is not necessary. Basis.js has `basis.object.merge` or `basis.object.extend`